testsuite/tests/typing-jkind-bounds/composite.ml

```diff
@@ -523,7 +523,7 @@ type 'a t : immutable_data = Flat | Nested of 'a t t
 Line 1, characters 0-52:
 1 | type 'a t : immutable_data = Flat | Nested of 'a t t
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Error: The kind of type "t" is value
+Error: The kind of type "t" is immutable_data
```

Comment:
This error message is pretty bad now. And now that we have normalization and subsumption, it's probably time to remove the "don't print on alpha" trick. (Though I could be wrong there.)